scrapyloganalyzer/__init__.py

```diff
@@ -204,4 +204,22 @@ def error_rate(self):
         :returns: an estimated lower bound of the true error rate
         :rtype: float
         """
-        return self.item_counts["FileError"] / (self.item_counts["File"] + self.item_counts["FileError"])
+        error_count = self.item_counts["FileError"]
+        if "invalid_json_count" in self.logparser["crawler_stats"]:
+            error_count += self.logparser["crawler_stats"]["invalid_json_count"]
+        return error_count / (self.item_counts["File"] + error_count)
+
+    @property
+    def drop_rate(self):
+        """
+        Return the rate between total items and dropped items.
+
+        :returns: the dropped items rate
+        :rtype: float
+        """
+        if "item_dropped_count" in self.logparser["crawler_stats"]:
+            return self.logparser["crawler_stats"]["item_dropped_count"] / (
+                self.logparser["crawler_stats"]["item_scraped_count"]
+                + self.logparser["crawler_stats"]["item_dropped_count"]
+            )
+        return 0.0
```

Comment:
A little easier to read.

```suggestion
        stats = self.logparser["crawler_stats"]
        if "item_dropped_count" in stats:
            return stats["item_dropped_count"] / (stats["item_scraped_count"] + stats["item_dropped_count"])
        return 0.0
```